### PR TITLE
Transition action: Get token from keychain

### PR DESCRIPTION
**Pull request by elmundio87**
Transitioning a ticket via the workflow wasn't working for me - after choosing a new status type for a ticket, the workflow would fail silently.

In the debugger the issue manifested itself via this error message;
```
[ERROR: action.script] Must supply ticketId, action, and token
```

Debugging the issue revealed that the `token` variable (required by `Jira.transition()`) was `undefined`, because the `args` variable only contained the  values for `ticketId` and `action`.

In this PR I have changed the way the `token` variable is set prior to invoking `Jira.transition()` - it is now set by the result of `keychain.find()`.

With this change in place on my fork, I'm now able to alter the status of a ticket without an issue.

### CODE REVIEW

File: lib/action.js [hunk 1]
```diff
@@ -72,7 +73,8 @@ switch(query) {
       .catch(console.log);
   break;
   case 'transition':
-    let [ticketId, action, token] = args;
+    let [ticketId, action] = args;
+    token = keychain.find()
```
Comment (steyep) [inline lib/action.js]: This is your user's basic auth token and it isn't necessary to pass it here. The token that is expected by the `.transition` method is the [`atl_token`](https://community.atlassian.com/t5/Answers-Developer-Questions/quot-atl-token-quot-in-URL/qaq-p/575268).

File: lib/action.js [hunk 2]
```diff
@@ -72,7 +73,8 @@ switch(query) {
...
     Jira.transition(ticketId, action, token);
```
Comment (steyep) [inline lib/action.js]: According to the article linked above, it doesn't seem like the `atl_token` is even necessary anymore. I can push out a fix for this in an upcoming release once I have more time to research/test but in the meantime you can bypass it by simply passing a non-empty string to the method:
```
Jira.transition(ticketId, action, ' ');
```

**Comment by elmundio87** [comment]
Thanks @steyep, I'll close this PR for now while you investigate the issue further.
